src/views/Predictions/Positions.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import SwiperCore, { Keyboard, Mousewheel } from 'swiper'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import { Box } from '@pancakeswap-libs/uikit'
+import { useGetSortedRounds } from 'state/hooks'
+import 'swiper/swiper.min.css'
+import RoundCard from './components/RoundCard'
+import Menu from './components/Menu'
+import useSwiper from './hooks/useSwiper'
+import usePollRoundData from './hooks/usePollRoundData'
+import useOnNextRound from './hooks/useOnNextRound'
+
+SwiperCore.use([Keyboard, Mousewheel])
+
+const StyledSwiper = styled.div`
+  .swiper-wrapper {
+    align-items: center;
+    display: flex;
+  }
+
+  .swiper-slide {
+    width: 320px;
+  }
+`
+const Positions: React.FC = () => {
+  const { setSwiper } = useSwiper()
+  const rounds = useGetSortedRounds()
+  const initialIndex = Math.floor(rounds.length / 2)
+
+  usePollRoundData()
+  useOnNextRound()
+
+  return (
+    <Box overflowX="hidden" overflowY="auto">
+      <Menu />
+      <StyledSwiper>
+        <Swiper
```

Comment:
This may be a case of personal preference, and I guess we'll get a better feel for this as more people use the carousel, but I think restricting movement to the arrow keys / nav buttons might be a shout? Performance and accuracy of click & drag / scroll isn't great.